diff --git a/node_modules/@types/apidoc/index.d.ts b/node_modules/@types/apidoc/index.d.ts
index 64b03e9..b6e7c7f 100755
--- a/node_modules/@types/apidoc/index.d.ts
+++ b/node_modules/@types/apidoc/index.d.ts
@@ -51,6 +51,7 @@ export interface DocOptions {
         [keys: string]: any;
     } | undefined;
     silent?: boolean | undefined;
+    dryRun?: boolean | undefined;
     simulate?: boolean | undefined;
     markdown?: boolean | undefined;
     lineEnding?: string | undefined;
@@ -59,4 +60,4 @@ export interface DocOptions {
     filterBy?: string | string[] | undefined;
 }
 
-export function createDoc(options: DocOptions): boolean | { data: Record<string, any>; project: Record<string, any> };
+export function createDoc(options: DocOptions): { data: Array<Record<string, any>>; project: Record<string, any> };
